### DISCUSSION TITLE
「想法」评论用户是否可在邮箱处填写webhook地址

### DISCUSSION THREAD

**Discussion by Jonnyan404** [ArtalkJS/Artalk#270] [Ideas]
### 使用场景：

由于现在各种机器人比较丰富，接收信息的渠道也比以往更多更方便快捷。

### 实现方式：

前端邮箱位置增加下拉框，选择填入 钉钉/飞书/微信/telegram 等机器人地址。可以复用针对管理员的各种通知逻辑。

**Comment by kecrily** [comment]
如果我是用户的话，我并不会选择这种通知方式，它太麻烦了而且不安全。大多数这些机器人通知方式都需要密钥，那么向站长提供密钥以获得（可能完全不重要的）通知提醒，风险和收益相差太大了。

**Comment by Jonnyan404** [comment]
> 如果我是用户的话，我并不会选择这种通知方式，它太麻烦了而且不安全。大多数这些机器人通知方式都需要密钥，那么向站长提供密钥以获得（可能完全不重要的）通知提醒，风险和收益相差太大了。

不管因为什么，你都有不选择这种方式的权利。
而我的提议，只是多一种可能性，仅此而已。
